api/openapi/openapiClient/docs/DefaultApi.md

```diff
@@ -0,0 +1,142 @@
+# \DefaultApi
```

Comment:
are we committing autogen doc in the repo?

Reply:
Will discuss this

Reply:
moved to spec folder.